src/esw/brushed_motors/ROSHandler.cpp

```diff
@@ -57,27 +73,82 @@ std::optional<float> ROSHandler::moveControllerOpenLoop(const std::string& name,
     return std::make_optional<float>(controller->getCurrentAngle());
 }
 
+// REQUIRES: nothing
+// MODIFIES: nothing
+// EFFECTS: Moves a controller in closed loop.
+std::optional<float> ROSHandler::moveControllerClosedLoop(const std::string& name, float targetAngle) {
+    auto controller_iter = ControllerMap::controllersByName.find(name);
+
+    if (controller_iter == ControllerMap::controllersByName.end()) {
+        return std::nullopt;
+    }
+
+    // TODO: actually write this code idk
+    Controller* controller = controller_iter->second;
+    controller->moveClosedLoop(targetAngle);
+
+    return std::make_optional<float>(controller->getCurrentAngle());
+}
```

Comment:
Okay the open loop and closed loop functions are nearly identical. Maybe it would be better to just make them one function called moveController and take a boolean for whether it's open or closed loop

Reply:
See previous comment

Reply:
If it's still being implemented, shouldn't it just be removed then